libs/components/src/lib/card/live-market/index.tsx

```diff
@@ -0,0 +1,108 @@
+import { qtMerge, Text, Button } from '@deriv/quill-design';
+import {
+  StandaloneChartTrendDownRegularIcon,
+  StandaloneChartTrendSidewayRegularIcon,
+  StandaloneChartTrendUpRegularIcon,
+} from '@deriv/quill-icons';
+import { LiveMarketContent } from '../types';
+import LivePrice from './live-price';
+
+export interface LiveMarketCardProps extends LiveMarketContent {
+  className?: string;
+}
+
+export const LiveMarketCard: React.FC<LiveMarketCardProps> = ({
+  className,
+  instrumentIcon,
+  instrument,
+  changePercentage,
+  status,
+  bidPrice,
+  askPrice,
+  spread,
+}) => {
+  const colorVariant = {
```

Comment:
please move these out of the component, these are not connected to any props or states, thanks 🙏